src/analysis/measure_arguments.Rmd

```diff
@@ -105,28 +93,29 @@ if (removed_outliers) {
 ```
 
 ```{r}
-extract_opcodes <- function(program_set_codename, arity) {
-  setwd("~/sources/imapp/gas-cost-estimator/src")
-  programs = read.csv(paste("../../local/pg_arguments_", program_set_codename, ".csv", sep=""))
-  programs = clean_programs(programs)
+extract_opcodes <- function(arity) {
   if (!missing(arity)) {
+    if (arity == 0) {
```

Comment:
Do I understand it correctly that it does not support DUPs and SWAPs as they has larger arity.

Reply:
DUPs and SWAPs are a bit strange. They have small arity (1 and 2 resp), but they skip elements of the stack. In the context of this analysis, they're treated as arity 1 and 2, the analysis doesn't "remember" the stack anymore.